src/org/usfirst/frc4904/robot/commands/Shoot.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.standard.commands.RunFor;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class Shoot extends CommandGroup {
```

Comment:
From an operator perspective, I want to be able to begin spinning up the flywheel while the driver is finishing alignment, so decoupling those actions lets me save cycle time. (To shoot automatically you can just run both Shoot and FlywheelSpinup in parallel.)